Gemfile

```diff
@@ -10,9 +10,6 @@ gemspec
 branch = ENV.fetch('SOLIDUS_BRANCH', 'master')
 gem 'solidus', github: 'solidusio/solidus', branch: branch
 
-# A workaround for https://github.com/bundler/bundler/issues/6677
-gem 'rails', '>0.a'
-
```

Comment:
One thing we've noticed on `solidus_taxjar` when we removed Rails from the `Gemfile` is that it exposed an issue upstream with `active_storage`. It is explicitly required [here](https://github.com/solidusio/solidus/blob/master/core/lib/spree/core.rb#L9) however it isn't listed as one of the Rials gems that are dependencies of `core` [here](https://github.com/solidusio/solidus/blob/master/core/solidus_core.gemspec#L26-L29). I still think this change is good, however if you see failures booting the dummy app, this could be the culprit. I believe this should be fixed upstream in core however.